src/Middleware/OutputCaching/src/OutputCachingOptions.cs

```diff
@@ -0,0 +1,49 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.ComponentModel;
+
+namespace Microsoft.AspNetCore.OutputCaching;
+
+/// <summary>
+/// Options for configuring the <see cref="OutputCachingMiddleware"/>.
+/// </summary>
+public class OutputCachingOptions
+{
+    /// <summary>
+    /// The size limit for the response cache middleware in bytes. The default is set to 100 MB.
```

Comment:
Update comment that refers to response caching (here and other places)

Reply:
Didn't find any other mention. The term "response cache" is used elsewhere but only to mention the store for cached responses.